code/modules/mob/living/emote.dm

```diff
@@ -423,6 +423,15 @@
 	key_third_person = "smugs"
 	message = "grins smugly."
 
+/datum/emote/living/snore
+	key = "snore"
+	key_third_person = "snores"
+	message = "snores."
+	message_mime = "sleeps soundly."
+	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
+	stat_allowed = UNCONSCIOUS
+	mob_type_blacklist_typecache = list(/mob/living/carbon/human)
+
```

Comment:
wait why did this get moved above sniff?
isn't sniff>snore alphabetical, rather than snore>sniff?

Reply:
no it's unintended